src/pages/console/formbuilder/customize.mdx

```diff
@@ -83,7 +83,7 @@ Spacing values can either be a CSS length value (`px`, `rem`, `em`, `%`) or a re
 
 ## Enable list inputs
 
-Form inputs can be made to accept multiple values. The form input gets "wrapped" around by a "list input" component to allow your users to add, update, and remove items. **This functionality is only enabled for inputs not tied to a data model.** If your input is tied to one of your existing models, this feature is controlled directly on the model by toggling the **Is array** property on the corresponding field and deploying your update.
+Form inputs can be made to accept multiple values. The form input gets **wrapped** around by a **list input** component to allow your users to add, update, and remove items. **This functionality is only enabled for inputs not tied to a data model.** If your input is tied to one of your existing models, this feature is controlled directly on the model by toggling the **Is array** property on the corresponding field and deploying your update.
```

Comment:
I think this is inaccurate. For data models with arrays, `Is array` is force checked and disabled. For ad-hoc forms, this is a selectable option.

Reply:
modified the text for accuracy